deno2/main.rs

```diff
@@ -0,0 +1,34 @@
+extern crate libc;
+use libc::c_char;
+use libc::c_int;
+use std::ffi::CStr;
+use std::ffi::CString;
+
+#[link(name = "deno", kind = "static")]
+extern "C" {
+    fn deno_v8_version() -> *const c_char;
+    fn deno_init();
+    fn deno_set_flags(argc: *mut c_int, argv: *const *const c_char);
```

Comment:
Also, AFAICT from the `deno_set_flags` implementation, it's actually `*mut *const c_char`.

Reply:
You mean `mut* mut* c_char`?
The c function signature is `void deno_set_flags(int* argc, char** argv);`